README.md

```diff
@@ -2,7 +2,9 @@
 
 | Windows Build Status |
 | -------------------- |
-| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager) |
+| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager?branch=master) |
+| [Latest Build](https://ci.appveyor.com/api/projects/xqemu-bot/xqemu-manager/artifacts/xqemu-manager.zip?branch=master) |
+https://ci.appveyor.com/api/projects/johnsmith/myproject/artifacts/bin/debug.zip?branch=master
```

Comment:
What is this for?

Reply:
Fat fingers, left some example code in the readme. Fixed.